opentelemetry-sdk/src/logs/concurrent_log_processor.rs

```diff
@@ -0,0 +1,53 @@
+use opentelemetry::InstrumentationScope;
+
+use crate::error::OTelSdkResult;
+
+use super::{LogBatch, LogExporter, LogProcessor, SdkLogRecord};
+
+/// A concurrent log processor calls exporter's export method on each emit. This
+/// processor does not buffer logs. Note: This invokes exporter's export method
+/// on the current thread without synchronization. i.e multiple export() calls
+/// can happen simultaneously from different threads. This is not a problem if
+/// the exporter is designed to handle that. As of now, exporters in the
+/// opentelemetry-rust project (stdout/otlp) are not thread-safe.
+/// This is intended to be used when exporting to operating system
+/// tracing facilities like Windows ETW, Linux TracePoints etc.
+#[derive(Debug)]
+pub struct SimpleConcurrentProcessor<T: LogExporter> {
```

Comment:
Should we name it as `SimpleConcurrentLogProcessor`  - to be consistent with `SimpleLogProcessor` and `BatchLogProcessor`.